allegroai/clearml
# add clearml ci/cd example

## Patch Description
![checks_green](https://user-images.githubusercontent.com/11781950/200600926-36fecf58-61b7-4d76-9116-d380b7f43825.png)
This change is to add 3 common/interesting CI/CD jobs that use ClearML, which can serve either as examples or as drop-in additions to an existing Github Actions workflow. The examples are implemented mainly in 3 python scripts, which the Github runner simply executes, so using these examples e.g. on Gitlab should not be much work.

## Testing Instructions
In order to run this workflow, copy paste the contents of this `cicd` example folder into a new git repository. Move `run_clearml_checks.yml` to a new folder `.github/workflows/run_clearml_checks.yml` so Github can pick it up. Then add your ClearML and optionally your Github credentials to the repository secrets using the keys specified above or in `run_clearml_checks.yml`. Finally, open a PR on the new repository with a change, and see how the Github Actions spring to life!

## Other Information
I have added a README with 2 images, but I can remove these if not wanted.
